products/PurePhone/assets.json

```diff
@@ -0,0 +1,74 @@
+{
+    "assets": [
+        {
+            "name": "./fonts/pure/dejavu_sans_bold_27.mpf",
+            "output":"assets/fonts/dejavu_sans_bold_27.mpf"
+        },
+        {
+            "name": "./fonts/pure/gt_pressura_regular_20.mpf",
+            "output":"assets/fonts/gt_pressura_regular_20.mpf"
```

Comment:
What about images and music? Will they be still in the repo?

Reply:
we require removal of this files due to licensing :) we can remove mp3 with - just add it to MuditaOSAssets and add to json description :)

Reply:
this should be pretty easy to do - we probably will do it in the nearest future if not now :P